website/docs/cloud-docs/integrations/aws-service-catalog/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+page_title: Terraform Cloud for AWS Service Catalog
```

Comment:
Question: Should we update this title to mirror the nav title [here](https://github.com/hashicorp/terraform-docs-common/pull/507/files#diff-db6bf0196608ec8d6db237317e526dca37469066eecb51be266577ac0dbcd554R583)? I recognize that this is the name of the GitHub repo, but it might be nice to generalize it in the docs a bit since it's for both TFC/E.
```suggestion
page_title: AWS Service Catalog Integration
```

Reply:
I think the current name matches the example set by the other integrations a little better. I'd have no objections to changing this title in a future PR though